### PR TITLE
Completed html-forms exercise.

### PR DESCRIPTION
**Pull request by sararoohan**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="http://www.example.com/" method="GET">
+    <div>
+      <label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you want a for attribute here and an id on the input

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
tiny update needed!

File: html-forms/index.html (hunk 1 above)
Comment (sararoohan) [reply html-forms/index.html]: Updated!

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**